package.json

```diff
@@ -29,40 +29,49 @@
     "Google Cloud Speech API"
   ],
   "scripts": {
-    "cover": "nyc --reporter=lcov mocha test/*.js && nyc report",
+    "cover": "nyc --reporter=lcov mocha build/test/*.js && nyc report",
     "docs": "jsdoc -c .jsdoc.js",
-    "lint": "eslint '**/*.js'",
+    "lint": "gts check && eslint '**/*.js'",
     "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test/*.js --timeout 600000",
-    "test-no-cover": "mocha test/*.js",
+    "system-test": "mocha build/system-test/*.js --timeout 600000",
+    "test-no-cover": "mocha build/test/*.js",
     "test": "npm run cover",
```

Comment:
can we just replace this with `c8 mocha build/test/*.js`, it seems like this has weird bespoke coverage logic (perhaps this was the first module we added coverage to).